CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Showing shipping estimate if has address postalCode;
```

Comment:
Nit: `.`

Reply:
If there is more than one topic I prefer to use the semicolon.